src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/InitializeDampedHarmonic.hpp

```diff
@@ -15,21 +15,130 @@
 #include "ErrorHandling/Error.hpp"
 #include "Evolution/Initialization/Tags.hpp"
 #include "Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DampedHarmonic.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/OptionTags.hpp"
 #include "Evolution/Systems/GeneralizedHarmonic/Tags.hpp"
+#include "Options/Options.hpp"
 #include "ParallelAlgorithms/Initialization/MergeIntoDataBox.hpp"
 #include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
 #include "Utilities/MakeWithValue.hpp"
 #include "Utilities/TMPL.hpp"
 
 namespace GeneralizedHarmonic::gauges::Actions {
+namespace InitializeDampedHarmonic_detail {
```

Comment:
why move the tags into a detail namespace? I would have anticipated that because option tags are more 'user-facing' than other parts, they should appear somewhere in the documentation even if the dox contents is minimal.

Reply:
Agreed. The part I'm undecided on is that `GeneralizedHarmonic::gauges::Actions` is very generic, and so are the names of the. I'll rename and make them visible. I am planning on combining all the options into a single struct so we have fewer compute tags (each ConstGlobalCache item results in a compute tag) in a later commit.